modules/transformer.js

```diff
@@ -1,14 +1,32 @@
 import parsers from './parsers/index.js';
+import CellUtils from './cells.js';
 
 /* global WebImporter */
 
 const IGNORE_ELEMENTS = [
   'style',
   'source',
   'script',
+  'noscript',
   'iframe',
```

Comment:
Be nice to add some concept about ignoring non-visible content (i.e. height: 0, x: 3000, etc.).  Some day.